ghost/admin/app/controllers/posts.js

```diff
@@ -85,11 +85,8 @@ export default class PostsController extends Controller {
         Object.assign(this, DEFAULT_QUERY_PARAMS.posts);
     }
 
-    get postsInfinityModel() {
-        return this.model;
-    }
-
     get totalPosts() {
+        // TODO: this might be a problem for the new infinity models
         return this.model.meta?.pagination?.total ?? 0;
     }
```

Comment:
This doesn't appear to be used anywhere, is it worth deleting it rather than having a comment that suggests it won't work?

Reply:
Happy to remove it - thanks for taking a pass!